docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,289 @@
+By defining a new manifest, registries and clients opt-into new capabilities, without breaking existing registry and client behavior or setting expectations for scenarios to function when the client and/or registry may not yet implement new capabilities
+
+### JSON Schema
+
+An Artifact manifest, representing a detached signature of the `hello-world@sha256:abc123` image:
+- The signature is stored as a `blob`
+- The signature points to the `hello-world@sha256:abc123` image through the `subject` property
+- Sorting of references may be achieved by setting a `created` annotation.
+
+```jsonc
+{
+  "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
+  "artifactType": "signature/example",
+  "blobs": [
+    {
+      "mediaType": "application/tar",
+      "digest": "sha256:def456",
+      "size": 32654
+    }
+  ],
+  "subject": {
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "digest": "sha256:abc123",
+    "size": 16724
+  },
+  "annotations": {
+    "org.opencontainers.artifact.created": "<createdDate>"
+  }
+}
+```
+
+### Manifest Properties
+
+- **`mediaType`** *string*
+
+  This field contains the `mediaType` of this document, differentiating from [image-manifest][oci-image-manifest-spec] and [image-index][oci-image-index]. The `mediaType` for this manifest type MUST be `application/vnd.oci.artifact.manifest.v1+json`, where the version WILL change to reflect newer versions.
+
+- **`artifactType`** *string*
+
+  The REQUIRED `artifactType` is a unique value, as registered with [iana.org][registering-iana].
+  Examples include `signature/example`, `sbom/example`, `ice-cream/example`.
+  For details on creating a unique `artifactType`, see [OCI Artifact Authors Guidance][oci-artifact-authors]
+
+- **`blobs`** *array of objects*
+
+  An OPTIONAL collection of 0 or more blobs. The blobs array is analogous to [oci.image.manifest layers][oci-image-manifest-spec-layers], however unlike [image-manifest][oci-image-manifest-spec], the ordering of blobs is specific to the artifact type. Some artifacts may choose an overlay of files, while other artifact types may store independent collections of files.
+    - Each item in the array MUST be an [artifact descriptor][descriptor], and MUST NOT refer to another `manifest` providing dependency closure.
+    - The max number of blobs is not defined, but MAY be limited by [distribution-spec][oci-distribution-spec] implementations.
+
+- **`subject`** *descriptor*
+
+   An OPTIONAL reference to any existing manifest within the repository. When specified, the artifact is said to be dependent upon the referenced `subject`.
+   - The item MUST be an [artifact descriptor][descriptor] representing a manifest. Descriptors to blobs are not supported. The registry MUST return a `400` response code when `subject` is not found in the same repository, and not a manifest.
```

Comment:
Requiring that the subject exist at the time the reference to it is created violates a couple of our requirements, doesn't it? Namely, being able to sign something before pushing the signed thing, and being able to delete a scanned thing but retain its scan report. I wonder if we should relax this to enable these use cases?

Reply:
See [Lifecycle Management](https://github.com/SteveLasker/wg-reference-types/blob/proposal-a/docs/proposals/PROPOSAL_A.md#lifecycle-management). It calls out you can leave the refType behind if you want upon parent artifact deletion. 

> Registries MAY treat the lifecycle of a reference type object, such as an SBoM or signature, as being tied to its subject. In such registries, when the subject is deleted or marked for garbage collection, the defined artifact is subject to deletion as well, unless the artifact is tagged.

On post, we did say the parent MUST exist, similar to layers must exist when a manifest is posted.

There are other ways to assure only signed content is pushed to a registry ([Quarantine Pattern](http://aka.ms/acr/quarantine)), and you could also the quarantine pattern to assure only scanned, validated SBoM content is pushed to a registry. 
This is one of the areas I removed to minimize the post here.